### DISCUSSION TITLE
BUG AO INSTALAR

### DISCUSSION THREAD

**Discussion by leandrofelix-dev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#338] [Q&A]
![Sem título](https://user-images.githubusercontent.com/58048297/115977236-79883800-a54c-11eb-904b-d698848ddc85.png)

**Comment by im-coder-lg** [comment]
Try building it in Command Prompt

**Reply by leandrofelix-dev** [reply]
even downloading the final version, do I still need to do this? I used it before formatting my notebook without any problems, but I formatted it and it was like this

**Reply by im-coder-lg** [reply]
I know, till the next release, you have to build it in CMD

**Comment by Melvin-Abraham** [comment]
Please refer to [this section of the Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-do-i-get-typeerror-error-processing-argument-at-index-0)
